modules/oe_webtools_captcha/config/install/oe_webtools_captcha.settings.yml

```diff
@@ -0,0 +1 @@
+validationEndpoint: 'https://europa.eu/webtools/rest/captcha/verify'
```

Comment:
As I understand from the initial plans in the ticket it should be configured through settings, not config?

Reply:
I discussed it with Hernani, it's be fine.